src/index.ts

```diff
@@ -0,0 +1,99 @@
+import { getDependencies } from './getdependencies';
+import { getRepoIds } from './getrepos';
+const JupiterOneClient = require('@jupiterone/jupiterone-client-nodejs');
+const gql = require("graphql-tag");
+
+const J1_USER_POOL_ID_PROD = "us-east-2_9fnMVHuxD";
+const J1_CLIENT_ID_PROD = "1hcv141pqth5f49df7o28ngq1u";
+const accessToken = '';
+
+let success = 0, failure = 0;
+const missingDeps = [];
+
+;(async () => {
+  const j1Client = await new JupiterOneClient({
+    account: 'j1dev',
+    username: 'j1dev',
+    password: 'dev',
+    poolId: J1_USER_POOL_ID_PROD,
+    clientId: J1_CLIENT_ID_PROD,
+    accessToken: accessToken,
+    dev: true
+  }).init();
+  const repoMap = await getRepoIds();
+  
+  //value = Repo Name, key = Repo Entity ID
+  for (let value of Array.from(repoMap.values())) {
```

Comment:
I'd personally love it if you'd break this file out. Right now it's a bit overwhelming! :dizzy_face: